source/api/presentation/3.0/index.md

```diff
@@ -483,15 +481,15 @@ The value _MUST_ be a string.
 
 The specific media type (often called a MIME type) for a content resource, for example `image/jpeg`. This is important for distinguishing different formats of the same overall type of resource, such as distinguishing text in XML from plain text.
 
+Please note that this is different to the `formats` property in the [Image API][image-api], which gives the extension to use within that API. It would be inappropriate to use in this case, as `format` can be used with any content resource, not just images.
```

Comment:
I would be inclined to get rid of the "Please", and start "Note that..." -- but don't feel very strongly on the issue